src/components/ReactionGameTopScoreTable.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+    <div class="high-scores">
+        <h2>Your High Scores</h2>
+        <p v-if="!reactionTimes.length">No high scores yet. Try playing a game!</p>
+        <table v-else>
+            <thead>
+                <tr>
+                    <th>Rank</th>
+                    <th>Date</th>
+                    <th>Score</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr v-for="(time, index) in sortedReactionTimes" :key="index">
+                    <td>{{ index + 1 }}</td>
+                    <td>{{ time.date }}</td>
+                    <td>{{ formatTime(time.score) }}</td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</template>
+
+
+<script>
+export default {
+    name: 'ReactionGameTopScoreTable',
+    data() {
+        return {
+            reactionTimes: JSON.parse(localStorage.getItem("reactionTimes")) || [],
+        };
+    },
+    computed: {
+        sortedReactionTimes() {
+            const sorted = [...this.reactionTimes].sort((a, b) => a.score - b.score);
+            return sorted.slice(0, 10);
+        },
+    },
+    methods: {
+        formatTime(score) {
+            const date = new Date(score);
+            return `${String(date.getUTCHours()).padStart(2, '0')}:${String(date.getUTCMinutes()).padStart(2, '0')}:${String(date.getUTCSeconds()).padStart(2, '0')}.${String(date.getUTCMilliseconds()).padStart(3, '0')}`;
+        },
+
+        // Fetch scores from localStorage
+        fetchScores() {
+            const savedScores = JSON.parse(localStorage.getItem('reactionTimes')) || [];
+
+            this.reactionTimes = savedScores;
```

Comment:
any reason for holding value in const?
yu can directly assign value to  this.reactionTimes